docs/loaders/pageloader.fsx

```diff
@@ -1,5 +1,4 @@
-#r "../_lib/Fornax.Core.dll"
-
+#r "nuget: Fornax.Core, 0.15.1"
```

Comment:
@xperiandri given that last version of this PR is targetting Fornax 0.16.x, I guess these refs need to be updated too? Also PR title.